tests/unit/insights/test_importer.py

```diff
@@ -278,23 +279,23 @@ def test_get_insight_update_annotated_references(self):
                 barcode=DEFAULT_BARCODE,
                 type=InsightType.label,
                 value_tag="tag1",
-                id=uuid.UUID("a6aa784b-4d39-4baa-a16c-b2f1c9dac9f9"),
                 annotation=-1,
             ),
             ProductInsight(
                 barcode=DEFAULT_BARCODE,
                 type=InsightType.label,
                 value_tag="tag2",
-                id=uuid.UUID("f3fca6c5-15be-4bd7-bd72-90c7abd2ed4c"),
```

Comment:
Oooh yes you got the courage to remove this :+1:

Reply:
yes it's a good opportunity to do some cleanup :wink: